### DISCUSSION TITLE
Which openmetadata-airflow-managed-apis release Can be successfully installed for MWAA？

### DISCUSSION THREAD

**Discussion by liujuanchi** [open-metadata/OpenMetadata#17964] [Q&A]
Hi there，

As the title，I failed to install openmetadata-airflow-managed-apis python package for AWS MWAA. I'm running MWAA 2.6.3 and Openmetadata 1.4.8, openmetadata-ingestion==1.4.8.0 was installed successfully. 
Could anyone help?

Thanks in advance!

**Comment by Prajwal214** [comment]
@liujuanchi you should be able install all the supported ingestion version in AWS MWAA

**Reply by liujuanchi** [reply]
@Prajwal214 thanks for that, but i'm trying to confire which one is the right thing:  openmetadata-managed-apis  or openmetadata-airflow-managed-apis?
from the doc I suppose the right one should be : pip install openmetadata-managed-apis==1.5.5 (in the readme.md it was openmetadata-airflow-managed-apis, given it's updating time is two year ago, it is definitely out of date.)

And the pypi page are also confusing, https://pypi.org/project/openmetadata-managed-apis/
It shows "pip install openmetadata-managed-apis" in the top, but "openmetadata-airflow-managed-apis" in middle of contents. 
In pypi page of https://pypi.org/project/openmetadata-airflow-managed-apis/, it's latest version is 0.11.5.3

I also try to install both by pip, and got following conflict: 

The conflict is caused by:
    openmetadata-managed-apis 1.5.5.0 depends on apache-airflow>=2.2.2
    openmetadata-airflow-managed-apis 0.11.5.3 depends on apache-airflow==2.1.4
